sam/models/sam_shap_explainer.py

```diff
@@ -0,0 +1,87 @@
+from typing import Callable
+
+import numpy as np
+import pandas as pd
+from sam.models import SamQuantileMLP
```

Comment:
There are many references still throughout the repo to `SamQuantileMLP`, which should be refactored to `TimeseriesMLP`

Reply:
> There are many references still throughout the repo to `SamQuantileMLP`, which should be refactored to `TimeseriesMLP`

Changed this. The docs files still need an update